index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="Abdul Ali Noori's portfolio" />
+    <link rel="stylesheet" href="styles.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+        <nav class="top-navigation">
+            <a href="#headline" class="welcome-logo">Welcome</a>
+            <img src="images/Icons/Icon-Menu.svg" alt="GitHub">
```

Comment:
- [ Optional] Please the `nav` tag represent navigational links it is not semantic to put the logo inside it since the logo is not a navigational link although we add a link to the home page in the logo.